src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/FindUsersByOrganisationIntegrationTest.java

```diff
@@ -52,9 +52,9 @@ public void retrieve_users_with_pending_organisation_status_should_return_no_use
     }
 
     @Test
-    public void retrieve_users_with_invalid_organisationIdentifier_should_return_status_404() {
+    public void retrieve_users_with_invalid_organisationIdentifier_should_return_status_400() {
         Map<String, Object> response = professionalReferenceDataClient.findUsersByOrganisation("123","False", hmctsAdmin);
-        assertThat(response.get("http_status")).isEqualTo("404");
+        assertThat(response.get("http_status")).isEqualTo("400");
 
     }
 
```

Comment:
do you have a separate test for 404? if an organisation is not found in DB... it should still throw 404

if an organisation name is invalid and fails validation then it should throw 400

Reply:
I agree, I will check if there is a test present already, if not I will create one.

Reply:
There is a test for 404 scenario named:

> retrieve_users_with_non_existing_organisationIdentifier_should_return_status_404